projects/plugins/protect/src/js/routes/scan/scan-toggle-group-control.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+	__experimentalToggleGroupControl as ToggleGroupControl, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption, // eslint-disable-line @wordpress/no-unsafe-wp-apis
+} from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { useCallback } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import useHistoryQuery from '../../data/scan/use-history-query';
+import useScanStatusQuery from '../../data/scan/use-scan-status-query';
+import styles from './styles.module.scss';
+
+/**
+ * ToggleGroupControl component for filtering threats by status.
+ *
+ * @return {JSX.Element|null} The component or null.
+ */
+export default function ScanToggleGroupControl(): JSX.Element {
+	const location = useLocation();
+	const navigate = useNavigate();
+	const { data: status } = useScanStatusQuery();
+	const { data: history } = useHistoryQuery();
+	const numActiveThreats = status ? status.threats.length : 0;
+	const numHistoricThreats = history ? history.threats.length : 0;
+
+	const selectedValue = location.pathname.includes( '/history' ) ? 'historic' : 'active';
+
+	const onChange = useCallback(
+		( value: string ) => {
```

Comment:
```suggestion
		( value: 'active' | 'historic' ) => {
```

Reply:
(IIRC, the threat status value used is "current", we could potentially use that here for consistency)

Reply:
>IIRC, the threat status value used is "current", we could potentially use that here for consistency

It is! I initially chose `active` as it first came up in the ToggleGroupControl task as `Active threats` was the toggle text, that then got passed along to keep things the same. Using `current` makes sense to me, what do think about updating the toggle text to `Current threats` also?